src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/AllTagsTests.java

```diff
@@ -37,7 +45,9 @@ public void prepareTest() {
     widgets.add(new TwitterWidgetPageObject(driver));
     widgets.add(new VKWidgetPageObject(driver));
     widgets.add(new WeiboWidgetPageObject(driver));
-    widgets.add(new GoogleFormWidgetPageObject(driver));
+
+    // Uncomment after 22-sep
+    //widgets.add(new GoogleFormWidgetPageObject(driver));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)